docs/contributors/develop.md

```diff
@@ -1,6 +1,6 @@
 # Developer Contributions
 
-Please also see [CONTRIBUTING.md](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md) for general information about contributions to the Gutenberg repository.
+Please also see [CONTRIBUTING.md](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/develop.md) for general information about contributions to the Gutenberg repository.
```

Comment:
This part reads `general information` but is placed under `Developer contributions`. 

So, I am not very sure if the previous link should be retained or if the change is good.

Reply:
It should stay as is. It’s correct as it refers to more general topics like licenses, documentation, etc.

Reply:
That's good to know. I have updated this in 7b85961d477513a4f81d0c9b40b3166faa4039e0.